### ISSUE TITLE
Experiment closure: nav on /new

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#8304]
## Description

In https://github.com/mozilla/bedrock/issues/8037 we built a Convert experiment that shows navigation on the /new page. It launched appx. 8am 2019-12-11. A week after that we should turn off the experiment, analyse its results, remove code artifacts, and file any follow ups.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] Experiment off
- [x] Code artifacts (/new/nav) removed
- [x] Analysis linked to this issue
- [ ] Follow up issues filed

**Comment by alexgibson** [comment]
The experiment is currently showing a drop off in conversions for the nav variation at -2%, with 99.50% confidence.

Is this enough data to confirm we should end this experiment?

**Comment by hoosteeno** [comment]
@sghoseWI the experiment has hit our time limit, but it has not hit the audience size we specified. Which should override here? Should we stop it now (time) or keep going a few days (sample size)?

https://app.convert.com/v#/projects/1003343/experiments_report/10032480
https://docs.google.com/document/d/1qQVvOPlz-E5ZRWrRUxFnzGsFwQ0mZ9rs_dTeT92P9VA/edit

**Comment by sghoseWI** [comment]
@hoosteeno the sample size threshold should override here. I think we will likely hit the 16,558 visitors per group this evening or tomorrow — so as soon as that happens we can feel free to shut the experiment down.

**Comment by alexgibson** [comment]
It looks like we've now had over 16,558 visitors per variation, which was specified in the doc. Are we ok to shut this off?

**Comment by alexgibson** [comment]
I've turned off the redirect worked that was diverting traffic to this experiment. @hoosteeno is going to turn the experiment off in Convert.

**Comment by hoosteeno** [comment]
It's turned off in Convert. @sghoseWI can you take a look at Convert's analysis, and run your own, and let us know your recommendation?

**Comment by sghoseWI** [comment]
@hoosteeno yes will do

**Comment by sghoseWI** [comment]
Experiment report here: https://docs.google.com/document/d/14MzAhKl9xI4ytf7D-g_3ciYdSlBWWchZSOoWc0zYFos/edit#

Tl;dr: Nav treatment underformed the control.  Recommending we keep the existing experience.

**Comment by dzingeek** [comment]
@sghoseWI is there any way we can get more data? 
- installs
- bounce rate
- was the nav download button counted in this?
- how did people get here? 

For me there are two very important pieces of data missing:
1. I think we should be looking at installs and not downloads as it's possible downloads went down but intent/installs went up. 
2. I also think we should look at where people came from...how they got here and what they did do once here?

**Comment by dzingeek** [comment]
Oh, I had one other question. What is the FxA section in the doc saying?

**Comment by sghoseWI** [comment]
@dzingeek For this experiment, we used Convert as our testing tool.  It gives us several advantages -- especially in terms of improving the efficiency and volume of tests we can run on the team... but it has some drawbacks as well.  We're quite a bit more limited in how much granular data we receive (i.e we cannot measure much about how people got to the pages like we can with a GA-based test -- and we cannot track installs).  The other thing to note is that when this test was designed, installs were not one of the success metrics we discussed tracking for this test.  In the future, if we want to track installs we will need to configure our test differently from the beginning (and put it in our test plan!).

We do have bounce rate data though -- and the treatment group outperformed the control by 2.1% (by decreasing the bounce rate).  My thinking was that this 2.1% improvement in bounce rate did not justify the statistically significant and large effect size magnitude decrease in download conversion rate.  And good catch on the FxA typo -- I corrected it to now say "Download Conversion Rate".

Justin might know something different -- but from what I understand, the experiment only tracked download conversions as measured by clicks on the main download button.  We did see just a 1% improvement in overall engagement -- as defined by when a visitor clicks on any link on the treatment page -- so it seems unlikely that there would be a meaningful enough impact on downloads from nav clicks to make up for the decrease we saw on the download button.  For what it's worth, in other analyses I've done, nav click downloads are usually trivially low relative to the main download CTA.

**Comment by hoosteeno** [comment]
> installs

As @sghoseWI mentions, we would need to plan for this when building the experiment. Measuring installs is non-trivial: requires substantially more design, coding & analysis.

>  was the nav download button counted in this?

It was hidden -- so, no.

> how did people get here?

A segment of people matching our experiment design were redirected at the CDN to a page at /exp/firefox/new/, which then split them and redirected each split to a page with or without nav. The population *should* resemble our normal population across other dimensions, e.g. source -- that's why we require minimum session counts. 

> it seems unlikely that there would be a meaningful enough impact on downloads from nav clicks to make up for the decrease we saw on the download button.

With some caveats about mixing and comparing data from different web analytics tools, we *can* measure the impact of the nav because users landed on unique URLs. So, for example, we can see that appx. 30 more users who saw the nav eventually downloaded on /firefox than those who did not see the nav. 

@sghoseWI , can you estimate (in the report, not here) the number of downloads this percentage decrease would represent if we put the nav on top for a quarter?

**Comment by sghoseWI** [comment]
@hoosteeno Thanks for the clarifications above.  Yes I will add that estimate to the report.

**Comment by hoosteeno** [comment]
Closing this, hoping for follow-up experiment plans.
